pkg/cmd/api/fields.go

```diff
@@ -136,6 +137,8 @@ func addParamsSlice(m map[string]interface{}, prevkey, newkey string) (map[strin
 				if lastMap, ok := lastItem.(map[string]interface{}); ok {
 					if _, keyExists := lastMap[newkey]; !keyExists {
 						return lastMap, nil
+					} else if reflect.TypeOf(lastMap[newkey]).Kind() == reflect.Slice {
```

Comment:
I've never written in Go before this weekend, so I'm unsure of the best way to do this.

What I'm trying to do it make sure `addParamsSlice` returns the previous nested object when adding a new item to an existing array in that nested object.